CHANGES.md

```
@@ -1,5 +1,9 @@
 # Changes
 
+## Version 0.3.6
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)